lib/src/services/jira_date_fns.ts

```diff
@@ -86,3 +95,43 @@ export const jiraFormattedDistance = (
   });
   return jiraFormattedDuration(duration);
 };
+
+/**
+ * Calculates the number of business hours (assumed 9-5, Mon-Fri) between two dates.
+ *
+ * @param to the 'end' date of the interval to measure.
+ * @param from the 'start' date of the interval to measure.
+ * @returns the number of business hours in the interval.
+ */
+export const differenceInBusinessHours = (
+  to: ReadonlyDate,
+  from: ReadonlyDate
+) => {
+  const businessDays = differenceInBusinessDays(to.getTime(), from.getTime());
+
+  return isBefore(to.getTime(), from.getTime())
+    ? 0
+    : (businessDays === 0
+        ? Math.max(
+            0,
+            differenceInMinutes(
+              min([to.getTime(), setMinutes(setHours(to.getTime(), 17), 0)]),
+              max([from.getTime(), setMinutes(setHours(from.getTime(), 9), 0)])
+            )
+          )
+        : Math.max(
+            0,
+            differenceInMinutes(
+              setMinutes(setHours(from.getTime(), 17), 0),
+              max([from.getTime(), setMinutes(setHours(from.getTime(), 9), 0)])
+            )
+          ) +
+          Math.max(
+            0,
+            differenceInMinutes(
+              min([to.getTime(), setMinutes(setHours(to.getTime(), 17), 0)]),
+              setMinutes(setHours(to.getTime(), 9), 0)
+            )
+          ) +
+          (businessDays - 1) * 8 * 60) / 60;
```

Comment:
Is 8 working hours an assumption that is made elsewhere in jiralint? Should we extract it? Does it interact badly with the 7.6 assumption from elsewhere?

Reply:
It would be nice to be able to feed this thing a list of public holidays 🤔

Reply:
I'll create a separate ticket to make these assumptions configurable.

I did think briefly about public holidays. I decided that the substandard technical implementation may make people more likely to want to finish work before an extended break.